package.json

```diff
@@ -31,9 +32,11 @@
     "test": "npm run build && mocha test",
```

Comment:
We need to add a test to verify that the types are generated correctly, like a TS file `testTypes.ts` that spins up a simple workerpool, and that we use to verify that it compiles without TS errors. What do you think?